<?xml version="1.0"?>
<launch>

  <!-- NOT USED-->
  <arg name="cartesian_limits" default="false" doc="not used"/> 
  <arg name="remove_table" default="false" doc="not used"/>
  <arg name="xyz_lower_limits" default="[-0.2, -0.2, 0.0]" doc="lower x,y,z cartesian limits on cloud"/>
  <arg name="xyz_upper_limits" default="[0.2, 0.2, 0.75]" doc="upper x,y,z cartesian limits on cloud"/>
  <arg name="point_cloud_topic" value="/zed2_node/point_cloud/cloud_registered" doc="not used"/>
  <!-- NOT USED-->
  
  <arg name="cloud_dir" value="$(find amiga_grasp)/data/tmp/" doc="directory to point clouds"/>

  <!-- Point cloud data collection -->
  <node name="point_cloud_server" pkg="moveit_task_constructor_gpd" type="point_cloud_server" output="screen">
    <!-- NOT USED-->
    <param name="cartesian_limits" value="$(arg cartesian_limits)"/>
    <rosparam param="xyz_lower_limits" subst_value="false">$(arg xyz_lower_limits)</rosparam>
    <rosparam param="xyz_upper_limits" subst_value="false">$(arg xyz_upper_limits)</rosparam>
    <param name="point_cloud_topic" value="$(arg point_cloud_topic)"/>
    <param name="remove_table" value="$(arg remove_table)"/>
    <!-- NOT USED-->
    
    <param name="cloud_dir" value="$(arg cloud_dir)" doc="only this is used"/>
  </node>
  
</launch>
